schema/field.go

```diff
@@ -52,8 +52,9 @@ type Constraints struct {
 	// represent null values.
 	Required bool `json:"required,omitempty"`
 
-	Maximum string `json:"maximum,omitempty"`
-	Minimum string `json:"minimum,omitempty"`
+	Maximum   string `json:"maximum,omitempty"`
+	Minimum   string `json:"minimum,omitempty"`
+	MinLength int    `json:"minlength,omitempty"`
```

Comment:
The json field should be called `minLength`, as per https://specs.frictionlessdata.io/table-schema/